lib/discourse_api/single_sign_on.rb

```diff
@@ -5,6 +5,14 @@
 
 module DiscourseApi
   class SingleSignOn
+    # We inherit from RuntimeError instead of StandardError for backward
+    # compatibility reasons. We used to raise RuntimeError but that was too
+    # generic and could be raised by libraries/code from other sources so we
+    # changed it to a custom error (ParseError) to make it possible to rescue
+    # exceptions that only relate to parsing errors
```

Comment:
Actually, can we add a test here instead of leaving a descirption?

Reply:
done!